win32/status.html

```diff
@@ -244,6 +244,19 @@ <h1 id="status-header">Agent info</h1>
             <dt>Queue length</dt>       <dd>{{ forwarder['queue_length'] }}</dd>
           </dl>
 
+          {% if forwarder['proxy_data'] %}
+          <h3>Proxy</h3>
+
+          <dl>
+            <dt>Host</dt>     <dd>{{ forwarder['proxy_data']['host'] }}</dd>
+            <dt>Port</dt>     <dd>{{ forwarder['proxy_data']['port'] }}</dd>
+            {% if forwarder['hidden_username'] %}
+            <dt>Username</dt>     <dd>{{ forwarder['hidden_username'] }}</dd>
```

Comment:
This variables "hidden_username" and "hidden_password" won't exist. They have to be set in the ForwarderStatus constructor.